### PR TITLE
clean outdated resources

### PR DESCRIPTION
**Pull request by andre-merzky**

### CODE REVIEW

File: src/radical/pilot/configs/resource_aliases.json [hunk 1]
```diff
@@ -1,33 +1,14 @@
 
 {
     "aliases": {
-        "das4.fs2"                          : "das4.fs2_ssh",
-        "das5.fs1"                          : "das5.fs1_ssh",
-        "epsrc.archer"                      : "epsrc.archer_orte",
         "fub.allegro"                       : "fub.allegro_rsh",
```
Comment (mturilli) [inline src/radical/pilot/configs/resource_aliases.json]: Is this current?

File: src/radical/pilot/configs/resource_aliases.json [hunk 2]
```diff
@@ -1,33 +1,14 @@
...
         "nersc.hopper"                      : "nersc.hopper_orte",
         "nersc.edison"                      : "nersc.edison_orte",
-        "ornl.titan"                        : "ornl.titan_orte",
-        "rice.davinci"                      : "rice.davinci_ssh",
-        "rice.biou"                         : "rice.biou_ssh",
-        "stfc.joule"                        : "stfc.joule_runjob",
         "xsede.stampede"                    : "xsede.stampede_ssh",
         "xsede.wrangler"                    : "xsede.wrangler_ssh",
-        "xsede.lonestar"                    : "xsede.lonestar_ssh",
         "xsede.trestles"                    : "xsede.trestles_ssh",
```
Comment (mturilli) [inline src/radical/pilot/configs/resource_aliases.json]: trestles is retired

File: src/radical/pilot/configs/resource_aliases.json [hunk 3]
```diff
@@ -1,33 +1,14 @@
...
-        "futuregrid.bravo"                  : "futuregrid.bravo_ssh",
-        "futuregrid.delta"                  : "futuregrid.delta_ssh",
-        "futuregrid.echo"                   : "futuregrid.echo_ssh",
-        "futuregrid.xray"                   : "futuregrid.xray_aprun",
-        "iu.bigred2"                        : "iu.bigred2_aprun",
-        "lrz.supermuc"                      : "lrz.supermuc_ssh",
-        "lumc.shark"                        : "lumc.shark_ssh",
-        "lumc.gb-ui"                        : "lumc.gb-ui_ssh",
         "ncar.yellowstone"                  : "ncar.yellowstone_ssh",
         "ncsa.bw"                           : "ncsa.bw_orte",
```
Comment (mturilli) [inline src/radical/pilot/configs/resource_aliases.json]: BW will be back as a cluster dedicated to climate image processing. Hopefully, its stack will change but even if it returns as it was, should we have bw_prrte?

File: src/radical/pilot/configs/resource_epsrc.json [hunk 4]
```diff
@@ -1,3 +1,4 @@
+
 {
     "archer_aprun": {
```
Comment (mturilli) [inline src/radical/pilot/configs/resource_epsrc.json]: Should we eliminate archer completely?

File: src/radical/pilot/configs/resource_aliases.json (hunk 1 above)
Comment (andre-merzky) [reply src/radical/pilot/configs/resource_aliases.json]: fs1 and 2 don't exist anymore, archer has changed completely by now, any configuration we have is outdated.

File: src/radical/pilot/configs/resource_aliases.json (hunk 2 above)
Comment (andre-merzky) [reply src/radical/pilot/configs/resource_aliases.json]: thanks, removed.

File: src/radical/pilot/configs/resource_aliases.json (hunk 3 above)
Comment (andre-merzky) [reply src/radical/pilot/configs/resource_aliases.json]: We should add a valid config once we can test it, but its probably not worthwhile including a config which we know won't work... :/  I'll remove the BW configs for now, ok?

File: src/radical/pilot/configs/resource_epsrc.json (hunk 4 above)
Comment (andre-merzky) [reply src/radical/pilot/configs/resource_epsrc.json]: done

**Pull request merged**